tools/uuids.js

```diff
@@ -12,14 +12,17 @@ function fetchScan () {
   return client.search({
     index: options.index,
     type: 'item',
-    search_type: 'scan',
     sort: [ '_doc' ],
     scroll: '1m',
-    size: 1000,
+    size: 10000,
     _source: false
   })
     .then((response) => {
       status.total = response.hits.total
+
+      response.hits.hits.forEach((item, i) => output.write(item._id + '\n'))
```

Comment:
Not sure if this is a 1.5 vs 5.1 thing, but I was just missing out on the initial uuids returned from the first search query.